playbook/workshops/remote_control/README.md

```diff
@@ -0,0 +1,615 @@
+
+Then add this code in the JavaScript
+
+```js
+Wand.threshold = 5;
+Wand.onWave = function() {
+  alert("Wand Waved");
+  // INSERT CODE HERE YOU WANT TO RUN WHEN YOU HAVE THE WAND
+};
+```
+
+> ![](img/wand_setup.gif)
+
+Adjust
+
+```js
+Wand.threshold = 5;
+```
+
+The larger the number, the harder you have to fling the wand in order for it to
+trigger.
+
+For an iPhone 6, I personally like
+```js
+Wand.threshold = 20;
+```
+
+**Important Note**
+
+You will need to open the output view of JS Bin on your phone. After you do that
+try waving your phone around.
+
+- [Here is a bare bones working example](http://jsbin.com/fawuda/121/edit?html,js)
+- [This is the output URL](http://jsbin.com/fawuda/121)
+
+Here is a more complicated example
+
+- [link to the code](http://jsbin.com/fawuda/128/edit?html,js,output)
+- [link to the output that you should open on your phone]
+  (http://jsbin.com/fawuda/128)
+
+**Important Note** that for waving the wand to play a sound, you need to press
+the button on your phone to play the sound first before the wand will work
+because of an iOS limitation ( a sound won't play until it is played at least
+once from pressing or clicking something). [Here is the output URL that you want
+to open on your phone.
+
+#### When you receive a text message
+
+```js
+Twilio.listenForMessages(function(data) {
+  alert("I received an SMS from " + data.from);
+  alert("They said " + data.body);
+});
+```
+
+[Here is an example JS Bin](http://jsbin.com/hisavu/4/edit?html,js,console)
+with the above code.
+
+These are all the properties that you can receive:
+
+```json
+{
+    "toCountry": "US",
+    "toState": "PA",
+    "smsMessageSid": "SMe182077e5d6045773c289815dcca25fd",
+    "numMedia": "0",
+    "toCity": "NORRISTOWN",
+    "fromZip": "19486",
+    "smsSid": "SMe182077e5d6045773c289815dcca25fd",
+    "fromState": "PA",
+    "smsStatus": "received",
+    "fromCity": "NORRISTOWN",
+    "body": "Yo",
+    "fromCountry": "US",
+    "to": "14846854323",
+    "toZip": "19406",
+    "numSegments": "1",
+    "messageSid": "SMe182077e5d6045773c289815dcca25fd",
+    "accountSid": "AC0a9665bd93b240f6ad384761b3e3b47c",
+    "from": "16107610083",
+    "apiVersion": "2010-04-01"
+}
+```
+
+And then you can do cool stuff like whenever you text the Twilio phone number,
+it will call you and say over the phone whatever you texted it. To find out the
+Twilio phone number to text, look in the JS Bin console.
+
+```js
+Twilio.listenForMessages(function(data) {
+  Twilio.sendMessage(data.from, "You said '" + data.body + "'");
+});
+```
+
+See the above code working in [this JS Bin](http://jsbin.com/hisavu/8/edit?html,js,console).
+
+```js
+Twilio.listenForMessages(function(message) {
+  Twilio.callAndPlay(message.from, "http://www.nyan.cat/music/original.mp3");
+});
+```
+
+See the above code working in [this JS Bin](http://jsbin.com/fawuda/111/edit?js,console).
+
+You can start to do more advanced logic like implementing a chat bot.
```

Comment:
They don't know control flow though :frowning:... Kidding ;)

Great read. I am really looking forward to seeing this in the hands of my club members!